.gitignore

```diff
@@ -24,4 +24,5 @@ runtimes
 /Samples/Magick.NET.Samples/Samples/Magick.NET/Files/*.pp
 /Samples/Magick.NET.Samples/Samples/Magick.NET/Scripts/*.pp
 /TestResults
-/Tools/AppVeyor/OpenCover*
\ No newline at end of file
+/Tools/AppVeyor/OpenCover*
+/ImageMagick/release

```

Comment:
Can you explain why this was added? I am pretty sure that this folder should not be created.